### ISSUE TITLE
Improve support for HTML5 Player

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4046]
**Reported by netblue on 2014-04-03 09:33**
Hi, when using the HTML5 player on websites NVDA identifies sum clickable elements as just clickable without description or identification of the element. Also volume controls cannot be accessed with NVDA but they can with Jaws.

**Comment by bhavyashah** [comment]
This ticket contains no specifics - no STR, no test case, and no means to contact the original reporter. Please consider closing as Invalid. @jcsteh @feerrenrut @leonardder

**Comment by feerrenrut** [comment]
Happy to reopen this issue if we can get some more specific information:

- [ ] Example website or more ideally a minimal test case.
- [ ] The actions required to trigger the problematic behaviour, or in other words "the steps to reproduce".
